core/src/main/java/io/grpc/internal/RetriableStream.java

```diff
@@ -203,21 +203,21 @@ private void commitAndRun(Substream winningSubstream) {
     }
   }
 
-  private Substream createSubstream(int previousAttemptCount) {
+  private Substream createSubstream(int previousAttemptCount, boolean isTransparentRetry) {
     Substream sub = new Substream(previousAttemptCount);
     // one tracer per substream
     final ClientStreamTracer bufferSizeTracer = new BufferSizeTracer(sub);
     ClientStreamTracer.Factory tracerFactory = new ClientStreamTracer.Factory() {
       @Override
-      public ClientStreamTracer newClientStreamTracer(
-          ClientStreamTracer.StreamInfo info, Metadata headers) {
+      public ClientStreamTracer newClientStreamTracer(ClientStreamTracer.StreamInfo info) {
         return bufferSizeTracer;
       }
     };
 
     Metadata newHeaders = updateHeaders(headers, previousAttemptCount);
+    // TODO: pass real substream id and transparentRetry flag
```

Comment:
Remove part of comment about transparentRetry flag?

Reply:
Removed.